### DISCUSSION TITLE
BigQuery support

### DISCUSSION THREAD

**Discussion by bob-rietveld** [rstudio/pointblank#404] [Ideas]
Hi, 

Thanks for creating pointblank, it is a next level R package, in terms of documentation, approach and execution. [ Hat tip]. 

I work in GCP environment, and was wondering if support for BigQuery (https://bigrquery.r-dbi.org/) is something you might consider.

Cheers,

Bob

**Comment by rich-iannone** [comment]
Hi and thanks for starting this discussion topic. I’d like to have better integration with BigQuery. It reportedly works (see the discussion here: https://github.com/rich-iannone/pointblank/discussions/244) but hasn’t been formally tested by me. 

I’m thinking of doing this but I also wouldn’t mind your help if you could also run a test suite as well. 

Would you open an issue so this could all be tracked? Then we could discuss more about what needs to be done in there!
